src/main/java/com/example/MainTask3.java

```diff
@@ -0,0 +1,18 @@
+//3.Вывести заданное количество случайных чисел с переходом и без перехода на новую строку.
+
+package com.example;
+
+import java.util.Scanner;
+
+public class MainTask3 {
+    public static void main(String[] args){
+        Scanner enterQuantity = new Scanner(System.in);
+        System.out.println("Enter the number of random numbers: ");
+        int quantity= enterQuantity.nextInt();
+        int a =0;
```

Comment:
The variable names `a` and `b` don't say us anything.
Please rename to e.g. `minRandomBound` and `maxRandomBound`.

But if you use this variables only in one place, just inline them and remove `a` at all:
```
for(int i = 0; i < quantity; i++) {
    System.out.println((int) (Math.random() * 10));
}
```